web/.storybook/config.ts

```diff
@@ -0,0 +1,20 @@
+import { configure, addDecorator } from '@storybook/react'
+import { ThemeDecorator } from './decorators/ThemeDecorator'
+
+// @ts-ignore
+window.env = {
+  REACT_APP_FIDER_LINK: 'https://psufiderdev.pathfinder.gov.bc.ca/',
+  REACT_APP_KEYCLOAK_AUTH_URL: 'https://sso-dev.pathfinder.gov.bc.ca/auth',
```

Comment:
isn't this the wrong URL? I think this should be oidc.gov.bc.ca

Reply:
Doesn't matter because it's for storybook - should be able to put any junk in ther eno?

Reply:
Okay will put some junk link